airgun/entities/subscription.py

```diff
@@ -195,17 +203,32 @@ class SubscriptionDetails(NavigateStep):
 
     Args:
         entity_name: name of Subscription
+        virt_who: Whether this is a virt who client subscription.
     """
     VIEW = SubscriptionDetailsView
 
     def prerequisite(self, *args, **kwargs):
         return self.navigate_to(self.obj, 'All')
 
     def am_i_here(self, *args, **kwargs):
-        subscription_name = kwargs.get('subscription')
+        subscription_name = kwargs.get('entity_name')
```

Comment:
is it still useful? By default we have am_i_here which checks for entity_name in breadcrumb. The only difference is default check is `self.view.breadcrumb.locations[1] == entity_name` but here it's `self.view.breadcrumb.locations[-1] == entity_name`.  Is the full path longer than 2 nodes?

Reply:
the full path is exactly 2 nodes for example :
Subscriptions > Red Hat Enterprise Linux Server, Premium (8 sockets) (Unlimited guests)

do not understand here what you mean

Reply:
I simply changed that as we are passing entity_name and not subscription

Reply:
Please drop this method completely then. Take a look at airgun/navigation.py::NavigateStep.am_i_here() - it already does the same; the reason this method was implemented - custom name `subscription` instead of generic `entity_name`. But as you've changed the name to default one, default navigatestep's am_i_here should work great.